src/app/[locale]/_utils/api-utils.ts

```diff
@@ -72,3 +88,58 @@ export const extractResponse = async <R>(
     throw toError(response.left);
   }
 };
+
+/**
+ * Retries a fetch request in case of timeout or response with a specific status code.
+ * @param maxRetries ( Default ENV API_MAX_RETRY ) - Maximum number of retry attempts for the request.
+ * @param statusCodeRetry ( Default 500 ) - HTTP status code that should trigger the retry.
+ * @param backOff ( Default 3000 ) - Waiting interval between retry attempts in milliseconds, this parameter represents the waiting time between each attempt with exponential growth.
+ * @returns A function that performs the fetch request with retries.
+ */
+
+export function retryingFetch(
+  maxRetries: number = API_MAX_RETRY,
+  statusCodeRetry: number = 500,
+  backOff: Millisecond = 3000 as Millisecond
+): (input: RequestInfo | URL, init?: RequestInit | undefined) => Promise<Response> {
+  // a fetch that can be aborted and that gets cancelled after fetchTimeoutMs
+  const abortableFetch = AbortableFetch(agent.getHttpFetch(process.env));
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  const timeoutFetch: (input: any, init?: RequestInit | undefined) => Promise<Response> = toFetch(
+    setFetchTimeout(API_TIMEOUT, abortableFetch)
+  );
+  // configure retry logic with default exponential backoff
+  // @see https://github.com/pagopa/io-ts-commons/blob/master/src/backoff.ts
+  // const exponentialBackoff = calculateExponentialBackoffInterval();
+  const retryLogic = withRetries<Error, Response>(
+    maxRetries,
+    calculateExponentialBackoffInterval(backOff)
```

Comment:
Before of this modification we were using the default value of [500](https://github.com/pagopa/io-ts-commons/blob/a2731fd0f73faaaae269123397d0df030812eff0/src/backoff.ts#L10) ms. Am I right? Why now we changed it to `3000`?

Reply:
Just, for testing, fixed, sorry